src/authorities/OverdueInstanceAuthority.ts

```diff
@@ -27,16 +28,14 @@ export default class OverdueInstanceAuthority {
         this.timer = setInterval(() => {
             OverdueInstanceAuthority.logger.debug('Running OverdueInstanceAuthority overdue alert check');
 
-            const instances: PS2AlertsInstanceInterface[] = this.instanceHandler.getAllInstances();
-
-            instances.forEach((instance: PS2AlertsInstanceInterface) => {
-                if (instance.overdue()) {
-                    try {
-                        OverdueInstanceAuthority.logger.warn(`Instance ${instance.instanceId} on world ${instance.world} is OVERDUE! Ending!`);
-                        void this.instanceHandler.endInstance(instance);
-                    } catch (err) {
-                        OverdueInstanceAuthority.logger.error(`Overdue instance ${instance.instanceId} was unable to be forcefully ended!`);
-                    }
+            _.filter(this.instanceHandler.getAllInstances(), (instance) => {
```

Comment:
You can just use the filter function of the array.
```suggestion
            this.instanceHandler.getAllInstances().filter((instance) => {
```

Reply:
Is that using lodash though?

Reply:
No, filter is a implemented in the standard js Array object(I assume it is an array).